hedyorg/hedy
# 📜 Improve error message when a student is in another class

This PR changes the error code and message for the cases in which a student in invited to class but is already added to another class. The new error code is 'student_in_another_class' and the error message is 'This student is in another class.'
Translations for English and Dutch are provided.

Fixes #5773

**How to test**
1. Log in as a teacher
2. Navigate to For Teachers > [an existing class] > Add students > Invite by username
3. Put in the username of a student that is already enrolled in another class and check that the error message appears
4. Click the 'Copy join link' button to get the invitation link
5. Log in as the student that is already in another class
6. Open the invitation link that you just copied and check that the same error message appears
